src/amo/i18n/utils.js

```diff
@@ -242,55 +241,43 @@ function oneLineTranslationString(translationKey) {
 }
 
 type FormatFilesizeParams = {|
-  _filesize?: typeof filesize,
-  _log?: typeof log,
   i18n: I18nType,
   size: number,
 |};
 
 // Translates a file size in bytes into a localized user-friendly format.
 export const formatFilesize = ({
-  _filesize = filesize,
-  _log = log,
   i18n,
   size,
 }: FormatFilesizeParams): string | null => {
```

Comment:
Maybe need to polyfill NumberFormat for older browsers.. https://caniuse.com/?search=NumberFormat